src/upnp_compat.h

```diff
@@ -0,0 +1,124 @@
+/* upnp_compat.h - libupnp v1.8.x/v1.6.x compatibilty layer
+ *
+ * Copyright (C) 2005-2007   Ivo Clarysse
```

Comment:
Given that this file is entirely contributed by you, I think you should write your copyright header here, 
  Copyright (C) 2019 Tucker Kern

Reply:
Don't want to be blamed for my mistakes? I get it. :)

Reply:
It is not even me :)
Ivo is the original author from whom I forked this project from.